src/globals.js

```diff
@@ -40,7 +40,7 @@ function joiner(sep) {
 function globals() {
     return {
         range: function(start, stop, step) {
-            if(!stop) {
+            if(!stop && stop !== 0) {
```

Comment:
Could we do `if(typeof stop === 'undefined')` here instead? Seems to communicate the actual intent more clearly.